src/APIs/quote-def.ts

```diff
@@ -0,0 +1,4 @@
+export interface quoteDef {
```

Comment:
Name interfaces similar to `class` declarations.

```suggestion
export interface QuoteDef {
```